framework/src/modules/validators/method.ts

```diff
@@ -208,9 +208,6 @@ export class ValidatorsMethod extends BaseMethod {
 		}
 
 		const validatorAccount = await validatorsSubStore.get(methodContext, validatorAddress);
-		if (!validatorAccount.blsKey.equals(INVALID_BLS_KEY)) {
-			return false;
-		}
```

Comment:
why is this removed?

Reply:
At some point, the requirement of disallowing updates of the BLS key was dropped.

Reply:
hm. Then it is not aligned with the specs in the LIP, but on the other hand it is not critical, because we also provide this [warning](https://github.com/LiskHQ/lips/blob/8c374e7649c48bcc8bf9015ca2ebc8272b72998d/proposals/lip-0044.md?plain=1#L426):

> Note: For blockchains build with the SDK v6, this function should be not be called except from the [Legacy module](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0050.md#register-keys) used on the Lisk Mainchain. The reason is that changing BLS keys is not supported by the (interoperability) protocol.